apps/address-service/domains/address/access/ActualizeAddressesService.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Generated by `createservice address.ActualizeAddressesService --type mutations 'addresses:[AddressWhereUniqueInput!]!'`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canActualizeAddresses ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return !!(user.isAdmin || user.isSupport)
+}
```

Comment:
how should it work? only in manual mode?

Reply:
For now - yes. Next I'll move logic to use it here and in cron task.

Reply:
Also, it may possible to add button within admin interface to refresh particular address.

Reply:
Just addes a button to admin interface
<img width="1525" alt="image" src="https://github.com/user-attachments/assets/e8fa2610-c0fe-4a90-ba71-a5478bd05936" />